install_config/syncing_groups_with_ldap.adoc

```diff
@@ -433,6 +437,155 @@ users:
 <1> The name of the Group as specified by the user-defined name mapping.
 ====
 
+[[rfc2307-with-error-tolerances]]
+=== RFC 2307 with User-Defined Error Tolerances
+
+By default, if groups being synced contain members whose entries that are outside of the
+scope defined in the member query, the group sync fails with an error like:
+
+----
+Error determining LDAP group membership for "<group>": membership lookup for user "<user>" in group "<group>" failed because of "search for entry with dn="<user-dn>" would search outside of the base dn specified (dn="<base-dn>")".
+----
+
+Most often, failures of this type indicated a mis-configured user query. However, if it
+is certain that the user query is properly configured, these errors can be tolerated and
+the out of scope members ignored.
+
+Similarly, when the group sync process fails to locate a member for a group, it fails
+outright with errors like:
+
+----
+Error determining LDAP group membership for "<group>": membership lookup for user "<user>" in group "<group>" failed because of "search for entry with base dn="<user-dn>" refers to a non-existent entry".
+
+Error determining LDAP group membership for "<group>": membership lookup for user "<user>" in group "<group>" failed because of "search for entry with base dn="<user-dn>" and filter "<filter>" did not return any results".
+----
+
+Failures of this type most often indicate mis-configured LDAP group configurations
+on the LDAP server. However, if it is certain that the group entries on the LDAP server
+have been configured correctly, these errors can be tolerated and the problematic
+members ignored.
```

Comment:
This often indicates a mis-configured `usersQuery`. However, in cases where the group contains member entries that are known to be missing, setting `tolerateMemberNotFoundErrors: true` will allow the group sync to continue.